src/Microsoft.Identity.Web/DownstreamApiSupport/IDownstreamWebApi.cs

```diff
@@ -0,0 +1,78 @@
+        public Task<HttpResponseMessage> CallWebApiForUserAsync(
+            string optionsInstanceName,
+            Action<CalledApiOptions>? calledApiOptionsOverride = null,
+            ClaimsPrincipal? user = null,
+            StringContent? content = null);
+
+        /// <summary>
+        /// Calls a Web API consuming JSon with some data and returns data.
+        /// </summary>
+        /// <typeparam name="TInput">Input type.</typeparam>
+        /// <typeparam name="TOutput">Output type.</typeparam>
+        /// <param name="optionsInstanceName">Name of the options instance describing the API. There can
+        /// be several configuration namedsections mapped to a <see cref="CalledApiOptions"/>,
+        /// each for one API. You can pass-in null, but in that case <paramref name="calledApiOptionsOverride"/>
+        /// need to be set.</param>
+        /// <param name="input">Input parameter to the API.</param>
+        /// <param name="calledApiOptionsOverride">Overides the options proposed in the configuration described
+        /// by <paramref name="optionsInstanceName"/>.</param>
+        /// <param name="user">[Optional] claims representing a user. This is useful in scenarios like Blazor
+        /// or Azure Signal R where the HttpContext is not available. In the other scenarios, the library
+        /// will find the user itself.</param>
+        /// <returns>The value returned by the API.</returns>
+        public Task<TOutput> CallWebApiForUserAsync<TInput, TOutput>(
+            string optionsInstanceName,
+            TInput input,
+            Action<CalledApiOptions>? calledApiOptionsOverride = null,
+            ClaimsPrincipal? user = null)
+            where TOutput : class;
+
+        /// <summary>
+        /// Calls the Web API for the app, with the required scopes.
+        /// </summary>
+        /// <param name="optionsInstanceName">Name of the options instance describing the API. There can
+        /// be several configuration namedsections mapped to a <see cref="CalledApiOptions"/>,
+        /// each for one API. You can pass-in null, but in that case <paramref name="calledApiOptionsOverride"/>
+        /// need to be set.</param>
+        /// <param name="calledApiOptionsOverride">Overides the options proposed in the configuration described
+        /// by <paramref name="optionsInstanceName"/>.</param>
+        /// <param name="content">Http content in the case where <see cref="CalledApiOptions.HttpMethod"/> is
+        /// <see cref="HttpMethod.Patch"/>, <see cref="HttpMethod.Post"/>, <see cref="HttpMethod.Put"/>.</param>
+        /// <returns>An <see cref="HttpResponseMessage"/> that the application will process.</returns>
+        public Task<HttpResponseMessage> CallWebApiForAppAsync(
+            string optionsInstanceName,
+            Action<CalledApiOptions>? calledApiOptionsOverride = null,
+            StringContent? content = null);
```

Comment:
Change `content` to  `requestContent` in all method signatures to be more descriptive?

Reply:
Done.